pgp_keys.asc

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Can we add a comment to this file as to it's origin and state?

Reply:
Addressed here: https://github.com/joscha/codecov-buildkite-plugin/pull/28/commits/f69e53c6009e1aedb346796f63387e65559fa35c